src/panels/lovelace/editor/card-editor/hui-card-preview.ts

```diff
@@ -66,13 +66,17 @@ export class HuiCardPreview extends HTMLElement {
       return;
     }
 
-    if (curConfig && configValue.type === curConfig.type) {
+    // in case of an error we always want to recreate the element
+    if (!this._error && curConfig && configValue.type === curConfig.type) {
       try {
-        this._element.setConfig(deepClone(configValue));
+        this._error = false;
+        this._element.setConfig(deepFreeze(configValue));
```

Comment:
We shouldn't deep freeze here. This is a passed in value to this component. So now rendering something with this component has the side-effect of freezing your object. We should freeze, but we should freeze in the component that owns the config (the parent I think in this case)